Makefile

```diff
@@ -391,6 +393,7 @@ $(FCITX5_QT): | deps
 PRODUCTION_PARAMETERS := -d:production -d:chronicles_sinks=textlines[stdout],textlines[nocolors,dynamic]
 
 $(STATUS_CLIENT_APPIMAGE): override RESOURCES_LAYOUT := $(PRODUCTION_PARAMETERS)
+$(STATUS_CLIENT_APPIMAGE): override IPFS_GATEWAY_URL_ARG := $(PROD_IPFS_GATEWAY_URL_ARG)
```

Comment:
Any specific reason why we are using the [`override` directive](https://www.gnu.org/software/make/manual/html_node/Override-Directive.html) here? Or are we just cargo-culting it?

Reply:
![image](https://user-images.githubusercontent.com/11926403/186230834-16d4d6ac-3cd5-4084-9571-f2933a59cd35.png)